### ISSUE TITLE
Critical Exception Processing Transaction

### ISSUE THREAD

**Issue by CaelorS** [hyperledger/besu#4250]
### Description
Hi, 

I have an issue sychronizing besu with the network. I launch an x-snap synchronisation, and here is what I get, after a few day. I'm stuck here... 

Any help would be appreciated. Thank ! 

```août 10 18:21:23 jeremy-NUC10i7FNK besu[42974]: 2022-08-10 18:21:23.149+02:00 | nioEventLoopGroup-3-7 | INFO  | SyncTargetManager | Found common ancestor with peer Peer 0x68dff20f06112b6ee8... PeerReputation 100, validated? true, disconnected? false at block 15295479
août 10 18:21:39 jeremy-NUC10i7FNK besu[42974]: 2022-08-10 18:21:39.922+02:00 | Timer-0 | INFO  | DNSResolver | Resolved 2409 nodes
août 10 18:21:41 jeremy-NUC10i7FNK besu[42974]: 2022-08-10 18:21:41.343+02:00 | Timer-0 | WARN  | RlpxAgent | Attempt to connect to peer with no listening port: enode://92ae467ca2719a0ca683d103580b78f3d0d0d58be0f8313ebe268da05e94ef0f2d91aa7834b0ec260e4b121a04a4bbf428d31d2b86bd34df3141ee56ccd63b01@xx.xx.xxx.xx:0?discport=30853
août 10 18:21:41 jeremy-NUC10i7FNK besu[42974]: 2022-08-10 18:21:41.737+02:00 | Timer-0 | INFO  | DNSTimerTask | Refreshing DNS records with enrtree://AKA3AM6LPBYEUDMVNU3BSVQJ5AD45Y7YPOHJLEF6W26QOE4VTUDPE@all.mainnet.ethdisco.net
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]: 2022-08-10 18:21:43.745+02:00 | EthScheduler-Services-5 (importBlock) | ERROR | MainnetTransactionProcessor | Critical Exception Processing Transaction
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]: org.hyperledger.besu.ethereum.trie.MerkleTrieException: Unable to load trie node value for hash 0xf83f41ea3851008549a6dc38424b5253c2edcf2a3e7135c160b84f614d5e3baf location 0x
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.trie.StoredNode.lambda$load$0(StoredNode.java:133)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.Optional.orElseThrow(Optional.java:408)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.trie.StoredNode.load(StoredNode.java:131)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.trie.StoredNode.accept(StoredNode.java:63)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.trie.StoredMerklePatriciaTrie.get(StoredMerklePatriciaTrie.java:119)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.trie.StoredMerklePatriciaTrie.get(StoredMerklePatriciaTrie.java:39)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.worldstate.DefaultMutableWorldState$WorldStateAccount.getStorageValue(DefaultMutableWorldState.java:304)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.worldstate.UpdateTrackingAccount.getStorageValue(UpdateTrackingAccount.java:213)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.worldstate.UpdateTrackingAccount.getStorageValue(UpdateTrackingAccount.java:213)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.worldstate.UpdateTrackingAccount.getStorageValue(UpdateTrackingAccount.java:213)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.worldstate.UpdateTrackingAccount.getStorageValue(UpdateTrackingAccount.java:213)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.worldstate.UpdateTrackingAccount.getStorageValue(UpdateTrackingAccount.java:213)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.worldstate.UpdateTrackingAccount.getStorageValue(UpdateTrackingAccount.java:213)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.operation.SLoadOperation.execute(SLoadOperation.java:61)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.EVM.lambda$executeNextOperation$0(EVM.java:87)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.tracing.OperationTracer.lambda$static$0(OperationTracer.java:27)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.EVM.executeNextOperation(EVM.java:81)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.EVM.runToHalt(EVM.java:74)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.processor.AbstractMessageProcessor.codeExecute(AbstractMessageProcessor.java:161)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.evm.processor.AbstractMessageProcessor.process(AbstractMessageProcessor.java:173)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.mainnet.MainnetTransactionProcessor.process(MainnetTransactionProcessor.java:485)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.mainnet.MainnetTransactionProcessor.processTransaction(MainnetTransactionProcessor.java:397)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.mainnet.AbstractBlockProcessor.processBlock(AbstractBlockProcessor.java:161)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.mainnet.AbstractBlockProcessor.processBlock(AbstractBlockProcessor.java:42)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.mainnet.BlockProcessor.processBlock(BlockProcessor.java:75)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.MainnetBlockValidator.processBlock(MainnetBlockValidator.java:158)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.MainnetBlockValidator.validateAndProcessBlock(MainnetBlockValidator.java:112)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.MainnetBlockValidator.validateAndProcessBlock(MainnetBlockValidator.java:71)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.mainnet.MainnetBlockImporter.importBlock(MainnetBlockImporter.java:44)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.core.BlockImporter.importBlock(BlockImporter.java:44)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.eth.sync.fullsync.FullImportBlockStep.accept(FullImportBlockStep.java:61)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.eth.sync.fullsync.FullImportBlockStep.accept(FullImportBlockStep.java:31)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.services.pipeline.CompleterStage.run(CompleterStage.java:37)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.services.pipeline.Pipeline.lambda$runWithErrorHandling$3(Pipeline.java:152)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.lang.Thread.run(Thread.java:829)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]: 2022-08-10 18:21:43.748+02:00 | EthScheduler-Services-5 (importBlock) | INFO  | AbstractBlockProcessor | Block processing error: transaction invalid 'INTERNAL_ERROR'. Block 0x654084b13b3ba3ce5d5ac99a8c55e365c691acd602666b77e3b34d38396fd66e Transaction 0x096f9b1750a614fe71c74a796af966857cd2cca94994b44b57465faa9a881ab7
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]: 2022-08-10 18:21:43.753+02:00 | EthScheduler-Services-5 (importBlock) | ERROR | MainnetBlockValidator | Error processing block. Block 15295480 (0x654084b13b3ba3ce5d5ac99a8c55e365c691acd602666b77e3b34d38396fd66e)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]: 2022-08-10 18:21:43.766+02:00 | EthScheduler-Services-5 (importBlock) | WARN  | PipelineChainDownloader | Invalid block detected. Disconnecting from sync target. Failed to import block: Invalid block at #15295480 (0x654084b13b3ba3ce5d5ac99a8c55e365c691acd602666b77e3b34d38396fd66e)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]: 2022-08-10 18:21:43.770+02:00 | EthScheduler-Services-5 (importBlock) | WARN  | PipelineChainDownloader | Chain download failed. Restarting after short delay.
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]: java.util.concurrent.CompletionException: org.hyperledger.besu.ethereum.eth.sync.tasks.exceptions.InvalidBlockException: Failed to import block: Invalid block at #15295480 (0x654084b13b3ba3ce5d5ac99a8c55e365c691acd602666b77e3b34d38396fd66e)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.CompletableFuture.encodeRelay(CompletableFuture.java:367)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.CompletableFuture.completeRelay(CompletableFuture.java:376)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.CompletableFuture$UniRelay.tryFire(CompletableFuture.java:1019)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.services.pipeline.Pipeline.abort(Pipeline.java:180)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.services.pipeline.Pipeline.lambda$runWithErrorHandling$3(Pipeline.java:159)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at java.base/java.lang.Thread.run(Thread.java:829)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]: Caused by: org.hyperledger.besu.ethereum.eth.sync.tasks.exceptions.InvalidBlockException: Failed to import block: Invalid block at #15295480 (0x654084b13b3ba3ce5d5ac99a8c55e365c691acd602666b77e3b34d38396fd66e)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.eth.sync.fullsync.FullImportBlockStep.accept(FullImportBlockStep.java:62)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.ethereum.eth.sync.fullsync.FullImportBlockStep.accept(FullImportBlockStep.java:31)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.services.pipeline.CompleterStage.run(CompleterStage.java:37)
août 10 18:21:43 jeremy-NUC10i7FNK besu[42974]:         at org.hyperledger.besu.services.pipeline.Pipeline.lambda$runWithErrorHandling$3(Pipeline.java:152)
```

### Versions (Add all that apply)
* Software version: besu/v22.4.4/linux-x86_64/openjdk-java-11
* Java version: openjdk 11.0.16 2022-07-19
OpenJDK Runtime Environment (build 11.0.16+8-post-Ubuntu-0ubuntu120.04)
OpenJDK 64-Bit Server VM (build 11.0.16+8-post-Ubuntu-0ubuntu120.04, mixed mode, sharing)

* OS Name & Version: DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.4 LTS"
NAME="Ubuntu"
VERSION="20.04.4 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.4 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

* Kernel Version: Linux jeremy-NUC10i7FNK 5.15.0-43-generic #46~20.04.1-Ubuntu SMP Thu Jul 14 15:20:17 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

* Cloud VM, type, size: physical computer 

### Additional Information (Add any of the following or anything else that may be relevant)
* Besu setup info - genesis file, config options
miner-enabled=false
graphql-http-enabled=false
sync-mode="X_SNAP"
pruning-enabled=false
rpc-ws-enabled=false
data-path="/home/jeremy/hyperledger-besu/besu"
rpc-http-enabled=true
network="MAINNET"

**Comment by iamhsk** [comment]
Hi @CaelorS, do you get the same exception when you use the latest besu release? (v. 22.7.0)

**Comment by iamhsk** [comment]
Seems like this is potentially a duplicate of #4079. We'll take a look and confirm whether that's the case.

**Comment by matkt** [comment]
potential duplicate. just a difference between the two tickets. this one is forest and the #4079 is bonsai

**Comment by non-fungible-nelson** [comment]
<img width="1724" alt="Screenshot_2022-10-26_at_12 40 48" src="https://user-images.githubusercontent.com/85905982/198113153-179b3885-4031-4b61-9938-6ead99f8c5a2.png">
Bumping with more logs.

**Comment by garyschulte** [comment]
closing as fixed by #4409
